src/components/Sidebar.js

```diff
@@ -0,0 +1,83 @@
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from "@mui/material";
+import React from "react";
+import DirectionsOutlinedIcon from "@mui/icons-material/DirectionsOutlined";
+import GridViewOutlinedIcon from "@mui/icons-material/GridViewOutlined";
+import WbSunnyOutlinedIcon from "@mui/icons-material/WbSunnyOutlined";
+import BookmarkBorderOutlinedIcon from "@mui/icons-material/BookmarkBorderOutlined";
+
+const Sidebar = () => {
+  return (
+    <Box
+      sx={{
+        bgcolor: "#F5F7F9",
+        minHeight: "100vh",
+        display: { xs: "none", sm: "flex" },
+        flexDirection: "column",
+        justifyContent: "space-between",
+        flex: "10%",
+        p: 1,
+      }}
+    >
+      <List>
+        <ListItem disablePadding sx={{ marginTop: 3 }}>
```

Comment:
I see lot of code being written again and again. Could you short this code?

Reply:
Resolved